openvpn-install.sh

```diff
@@ -133,6 +133,7 @@ if [[ -e /etc/openvpn/server.conf ]]; then
 			rm -rf pki/issued/$CLIENT.crt
 			rm -rf /etc/openvpn/crl.pem
 			cp /etc/openvpn/easy-rsa/pki/crl.pem /etc/openvpn/crl.pem
+			chmod nobody:nobody /etc/openvpn/crl.pem
```

Comment:
1) Much better!
2) I'm an idiot - the irony is on my local (not git) script I actually did use chown...

Excuse my newbness - do I change my patch, or do you just submit your own? how does this work?

Thanks!